week2/2-website/index.html

```diff
@@ -1,21 +1,122 @@
+                    <li class="item">Store</li>
+                    <li class="item">Blog</li>
+                    <li class="item">Help</li>
+                    <li class="item">Login</li>
+                </ul>
+            </div>
+
+        </header>
+
+
+
+        <!-- Main body content -->
+        <div class="top_content">
+            <div class="top_content_container">
+                <h1>Introducing Karma</h1>
+                <h3>Bring Wifi with you, everywhere you go.</h3>
+                <div class="btn"><button>Learn More</button></div>
+            </div>
+
+        </div>
+        <div class="bottom_content">
+
+            <h2>Everyone needs a little karma.</h2>
+            <ul class="case">
+                <div>
+                    <li class="case_list"><img width="127" height="127" src="img/icon-devices.svg"
+                            alt="Internet for all devices image"></li>
+                    <div class="caption">
+                        <h5>Internet for all devices</h5>
+                    </div>
+                </div>
+                <div>
+                    <li class="case_list"><img width="127" height="127" src="img/icon-coffee.svg"
+                            alt="Boost your productivity image"></li>
+                    <div class="caption">
+                        <h5>Boost your productivity</h5>
+                    </div>
+                </div>
+
+                <div>
+                    <li class="case_list"><img width="127" height="127" src="img/icon-refill.svg"
+                            alt="Pay as You Go image">
+                    </li>
+                    <div class="caption">
+                        <h5>Pay as you go</h5>
+                    </div>
+                </div>
+
+            </ul>
+
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
 
+        </div>
+
+        <!-- Footer content -->
+        <hr>
+
+        <div class="footer">
+
+
+            <div>
+                <h5>Join us on</h5>
+            </div>
+
+
+
+            <ul class="icons">
+                <li class="twitter">
+                    <i class="fa fa-twitter"></i>
+                </li>
+                <li class="facebook">
+                    <i class="fa fa-facebook"></i>
+                </li>
+                <li class="instagram">
+                    <i class="fa fa-instagram"></i>
+                </li>
+            </ul>
+
+
+
+
+            <!--div><ul class="icons">
+                    <li class="twitter"></li><i class="fa fa-twitter"></i>
+                    <li class="facebook"></li><i class="fa fa-facebook"></i>
+                    <li class="instagram"></li><i class="fa fa-instagram"></i>
+                </ul></div-->
+
+            <div> © Karma Mobility, Inc.</div>
+
+        </div>
+    </div>
 </body>
-</html>
+
+</html>
```

Comment:
FYI 
Formatting and indentation could be improved throughout this file.
Sometimes it is easier to create large spaces between elements as you are developing - but you should remove these before you submit your work as it is quite difficult to read